### ISSUE TITLE
WebCalendar v1.3.0: Adding entries hardly possible without "+" button on touch devices

### ISSUE THREAD

**Issue by codifex** [craigk5n/webcalendar#140]
In previous versions every day in month view contained a little "+" button in the upper right corner. By clicking this button, the UI used to add a new appointment was opened.

However, in v1.3.0, it seems that this button has been removed. New appointments now can be added by double-clicking in the empty area of the day's box in month view.

This poses two problems:
+ On touch devices such as the iPhone double-clicks do not work and the only way to add a new entry is via the menu, which requires seleting the date (very clumsy on small devices).
+ In addition, the double-click solution requires one additional click compared to one click on the "+" button.

It would be great if the "+" button could be restored as it would significantly improve usability and accessibility.

P.S.: Despite my bug reports about v1.3.0 I am very glad about the recent effort to update this project and I would like to thank Craig all others who contributed  :-).

**Comment by blueskies76** [comment]
Yes, I would like to request for  "+" button in the upper right corner. Otherwise, it is very add to add new event on phone and tablets. I would support adding this feature

**Comment by DomoticX** [comment]
Same here!, with v1.3.0, we cannot add appointments on mobile devices (double tap zooms in), please make option to enable + sign again...downgrading to 1.2.9 for now...

**Comment by Fjorstut** [comment]
Recently upgraded from 1.2.7 to 1.3.0. I support the request to find a more optimal solution for adding new events on smartphone devices, which generally do not have a double-click functionality implemented, and if it has, it is commonly used to zoom the screen in and out.
